cove_360/templates/cove_360/additional_checks.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
```

Comment:
Given that this template was missing in a previous commit and the tests passed anyway, I wonder if we should have a simple test checking that this page is rendered correctly.